### PR TITLE
feat: Add loading indicator for staked only when user data is loaded

### PR DESCRIPTION
**Pull request by memoyil**
To review:

https://deploy-preview-1752--pancakeswap-dev.netlify.app/

To reproduce:

1. Connect your account
2. Go to farms or pools
3. Select staked only fast
4. See there is no loading indicator

### CODE REVIEW

File: src/components/Loading.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const Loading: React.FC = () => <StyledLoading />
```
Comment (hachiojidev) [inline src/components/Loading.tsx:4]: I don't understand why this is here.

Comment (memoyil) [reply src/components/Loading.tsx:4]: you mean the decleration?

Comment (hachiojidev) [reply src/components/Loading.tsx:4]: Yes, why not use the styled component directly?

Comment (memoyil) [reply src/components/Loading.tsx]: Done

**Review by hachiojidev — APPROVED** [review]

**Pull request merged**